cdm/core/src/main/java/ucar/nc2/filter/Classifier.java

```diff
@@ -30,6 +36,28 @@ public static Classifier emptyClassifier() {
   /** Enough of a constructor */
   public Classifier() {}
```

Comment:
I think you can get rid of the empty constructor now

Reply:
I changed it, but kept one in case the CreateFromVat fails, it will spawn a classifier with no arguments. Not sure if correct approach!